pkgdb2/templates/package.html

```diff
@@ -68,26 +68,55 @@
+                {%- if package.monitoring_status == False and g.fas_user %}
+                    class="active"
+                {%- elif package.monitoring_status == False %}
+                    class="active mouse_inactive"
+                {%- endif -%}>
+                No monitoring
+              </label> <br />
             <input type="radio" id="monitoring" name="monitoring" value="1" />
-                <label for="monitoring" {% if('packager' in g.fas_user.groups or is_admin)
-                    %} title="Turns on monitoring and attempt to build new releases for this package"{%
-                    elif g.fas_user %} title="Only package admin or owner can turn on monitoring and attempt to build new releases"{%
-                    else %} title="Login to turn on monitoring and attempt to build new releases"{% endif %}{%-
-                    if package.monitoring_status == True %} class="active"{% endif
-                    -%}>Bugs &amp; builds</label> <br />
+              <label for="monitoring" {%
+                if('packager' in g.fas_user.groups or is_admin)
+                %} title="Turns on monitoring and attempt to build new releases for this package"{%
+                elif g.fas_user
+                %} title="Only package admin or owner can turn on monitoring and attempt to build new releases"{%
+                else
+                %} title="Login to turn on monitoring and attempt to build new releases"
+                {%- endif %}
+                {%- if package.monitoring_status == True and g.fas_user %}
+                    class="active"
+                {%- elif package.monitoring_status == True %}
+                    class="active mouse_inactive"
+                {% endif-%}>
+                Bugs &amp; builds
+              </label> <br />
             <input type="radio" id="no_build" name="monitoring" value="nobuild" />
-                <label for="no_build" {%- if ('packager' in g.fas_user.groups or is_admin)
-                        %} title="Turns on monitoring of new releases but without attempt to build it"{%
-                        elif g.fas_user %} title="Only package admin or owner can turn on monitoring new releases"{%
-                        else %} title="Login to turn on monitoring of new releases for the package without attempting to build it"{% endif -%}{%-
-                    if package.monitoring_status == "nobuild" %} class="active"{% endif
-                    -%}>Bugs only</label>
+              <label for="no_build" {%-
+                if ('packager' in g.fas_user.groups or is_admin)
+                %} title="Turns on monitoring of new releases but without attempt to build it"{%
+                elif g.fas_user
+                %} title="Only package admin or owner can turn on monitoring new releases"{%
+                else
+                %} title="Login to turn on monitoring of new releases for the package without attempting to build it"
+                class="mouse_inactive"
```

Comment:
Looks like we forgot a ``class=`` too many, this will make the element has two ``class=`` which is not allowed. So let's remove this one :)